### DISCUSSION TITLE
Image Generator not supporting 'webp' format

### DISCUSSION THREAD

**Discussion by CSenshi** [spatie/laravel-medialibrary#2506] [Ideas]
why ImageGenerator class does not support `image/webp` mime type? See [supported mime type](https://github.com/spatie/laravel-medialibrary/blob/232f347ef7ea9f373d00fbc6280e6fb9489330ec/src/Conversions/ImageGenerators/Image.php#L27)

This is problem because, whenever use uploads image in `webp` format package can not generate responsive images because of [this check](https://github.com/spatie/laravel-medialibrary/blob/232f347ef7ea9f373d00fbc6280e6fb9489330ec/src/MediaCollections/HtmlableMedia.php#L48).

Any particular reason? Is there anything I can do?

**Comment by freekmurze** [comment]
Feel free to PR improvements. Make sure to add tests and update the docs.
